tests/integration/element__click_test.py

```diff
@@ -0,0 +1,35 @@
+from tests.integration.helpers.givenpage import GivenPage
+
+
+def test_click_waits_for_covering_preload_element_to_disappear(session_browser):
+    page = GivenPage(session_browser.driver)
+    page.opened_with_body(
+        ''' <style>
+             canvas {
+              background-color: rgba(0, 0, 0, 0.336);
+              position: fixed;
+              height: 100%;
+              width: 100%;
+              top: 0;
+              left: 0;
+              z-index: 1000;
+             }
+            </style>
+
+            <a href="#second">go to Heading 2</a>
+            <h2 id="second">Heading 2</h2>       
+
+            <p><canvas></canvas></p>'''
+    )
+    page.load_body_with_timeout(
+        '''
+        <a href="#second">go to Heading 2</a>
+        <h2 id="second">Heading 2</h2>       
+
+        ''',
+        250,
+    )
+
+    session_browser.element('a').click()
```

Comment:
this code will wait for 4seconds by default, this is too much... If for some reason something goes wrong, we will wait for 4 seconds in this test... so it would be better to minimize timeout, for example like this:
`session_browser.element('a').with_(timeout=0.5).click()`
or, in the very beginning of the test, start with:

```python
    browser = session_browser.with_(timeout=0.6)
    page = GivenPage(browser.driver)
```
And below use

```
        browser.element('a').click()
```